src/appleseed-max-impl/appleseedoslplugin/oslshadermetadata.cpp

```diff
@@ -102,6 +102,7 @@ namespace
 OSLParamInfo::OSLParamInfo(const asf::Dictionary& paramInfo)
   : hasDefault(false)
   , divider(false)
+  , lockGeom(true)
```

Comment:
I think we should change `OSLParamInfo` to use the same naming conventions as the rest of the project (`m_` prefix and snake_case for member variables).